From 8da8197608a9f1e993f8ab217c01026d6619f4b8 Mon Sep 17 00:00:00 2001
From: Mike Gerdts <mike.gerdts@joyent.com>
Date: Fri, 18 May 2018 16:10:58 +0000
Subject: [PATCH] OS-6963 add_fbuf should also use is_env_string (fix debug)
 Reviewed by: Jason King <jason.king@joyent.com> Reviewed by: Jerry Jelinek
 <jerry.jelinek@joyent.com> Approved by: Jerry Jelinek
 <jerry.jelinek@joyent.com>

---
 usr/src/lib/brand/bhyve/zone/boot.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/usr/src/lib/brand/bhyve/zone/boot.c b/usr/src/lib/brand/bhyve/zone/boot.c
index c6431e61f6..e29616a748 100644
--- a/usr/src/lib/brand/bhyve/zone/boot.c
+++ b/usr/src/lib/brand/bhyve/zone/boot.c
@@ -479,7 +479,6 @@ add_bhyve_extra_opts(int *argc, char **argv)
 static int
 add_fbuf(int *argc, char **argv)
 {
-	char *val;
 	char conf[MAXPATHLEN];
 	int len;
 
-- 
2.21.0

